### DISCUSSION TITLE
How to use with only one pin?

### DISCUSSION THREAD

**Discussion by FreeLANMan** [schreibfaul1/ESP32-audioI2S#509] [General]
I want to use this library with the TTGO VGA32 board and a project already in progress (PCEmulator Plus) and I only have 1 pin available (25). How can I configure this in my code?
I used 'Audio audio(true, I2S_DAC_CHANNEL_RIGHT_EN);' and it wasn't enough.

And what are the audio characteristics of an MP3 file to be used in this context?
channels
sampling rate
bits per sample
bitrate

**Comment by schreibfaul1** [comment]
This is exactly how it works.
The values ​​for the constructor with internal DAC on pin25 are:  ` Audio audio(true, 1, 0);`

channels  1/2 (mono/stereo)
samplerate e.g. 48000 (nr. of samples in one second)
bps can be 8 or 16
bitrate is the data transfer rate
